index.html

```diff
@@ -46,6 +196,35 @@ <h1>I'm Shaddai,<br />Glad to see you</h1>
             <a href="#"><img src="./img/Vector (3).svg" alt="logo" /></a>
           </li>
         </ul>
+        <a href="#" class="btn">Get my resume</a>
+        <ul class="about-technologies">
+          <li>
+            <span>Languages</span>
+            <img class="arrow" src="./img/arrow down.svg" alt="arrow down" />
+            <div>
+              <div class="lang-container">
+                <img src="./img/css logo.svg" alt="" />
+                <p>javascript</p>
+              </div>
+              <div class="lang-container">
+                <img src="./img/html logo.svg" alt="" />
+                <p>HTML</p>
+              </div>
+              <div class="lang-container">
+                <img src="./img/javascript logo.svg" alt="" />
+                <p>Css</p>
+              </div>
+            </div>
```

Comment:
Please consider wrapping the technologies in this section in a `ul` tag instead of a `div` and each technology item in a `li` tag instead of a `div` as mentioned in the project requirements. please see the screenshot below :point_down:  for more details.
![techstack](https://user-images.githubusercontent.com/16392046/191827759-14a01d53-6d09-4649-b81b-eb8ef8351865.png)

Reply:
ok sir, I'll do just that